src/01_bignum.py

```diff
@@ -1,4 +1,6 @@
 # Print out 2 to the 65536 power
 # (try doing the same thing in the JS console and see what it outputs)
 
-# YOUR CODE HERE
\ No newline at end of file
+# YOUR CODE HERE
+
+2**65536
```

Comment:
While this is correct, you want to show that you can use the print function. So you would put: 
print(2**65536)